content/aips/COLLECTOR-V2-ASSET-CONSOLIDATION.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Ethereum v2 Collector Contract Consolidation
+author: Llama, DeFi_Consulting, efecarranza, dydymoon
+shortDescription: Consolidate the assets held within the v2 Collector Contract
+discussion: https://governance.aave.com/t/arfc-ethereum-v2-collector-contract-consolidation/10909
+created: 2023-01-17
+---
+
+Abstract
```

Comment:
You seem to be missing the Markdown heading notation here and below. (i.e `#`)

Reply:
thanks